ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -873,5 +873,40 @@
     "CaseEventID": "manageRepresentatives-PREPARE_FOR_HEARING",
     "UserRole": "caseworker-publiclaw-courtadmin",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-courtadmin",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-gatekeeper",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "allocatedJudge",
+    "UserRole": "caseworker-publiclaw-solicitor",
+    "CRUD": "R"
+  },
```

Comment:
does this propagate to LASOLICITOR and SOLICITOR roles?

Reply:
Yes this lets them view the 'History' of events in LASOLICITOR and SOLICITOR because if we just give allocatedJudge a R event with [LASOLICITOR] permission they will not see the history tab - it's a ccd restriction I found when implementing case roles which is why we have 'R' permissions on every event for caseworker-publiclaw-solicitor